handbook/company/mission.md

```diff
@@ -6,51 +6,25 @@ mission: Learn more about the mission of Tuist
 
-- What is the average build time of a target?
-- Which test has been the most flaky in the past week?
-- What is the bottleneck slowing down compilation times?
-- Which output artifact has room for optimization?
-
-We believe that **having answers to these questions early on allows teams to steer projects in the right direction, fostering a thriving and productive development environment**. Teams should be proactive about addressing these areas, rather than reacting once issues negatively impact the team.
-
-Tuist plays a crucial role in collecting, processing, and presenting data in a meaningful way for developers and organizations. **We believe data should be actionable, not just informative.** Our goal is to provide insights that help developers and organizations make informed decisions about their projects.
-
-## Integrations
-
-Although teams spend much of their time within Apple's tools, they are part of larger organizations and ecosystems that include complementary tools, such as communication platforms like [Slack](https://slack.com/), version control systems like [GitHub](https://github.com), and telemetry tools like [Prometheus](https://prometheus.io/). In other ecosystems, such as web development, tools seamlessly integrate with these platforms, providing developers and leadership with insights to make informed decisions. However, the Apple ecosystem is often siloed within Xcode (e.g., [Xcode Cloud](https://developer.apple.com/xcode-cloud/)), which we believe is a missed opportunity.
-
-**Tuist aims to make it extremely easy to integrate projects and tools with other services.** Integration should be as simple as taking a few steps through a web interface. Any member of the organization should be able to interface with Tuist using only a web browser. We strive to meet every member of an organization where they are, without requiring them to install Apple's tools and translating the information into their preferred language (e.g., business).
-
-## Resources
-
-Many resources provided by Apple or available in the community focus on first-time experiences with technologies, often lacking the depth needed to address the challenges developers face as they scale their projects. Tuist is uniquely positioned to learn about these challenges and provide organizations with the best resources to overcome them. Every interaction with a developer is an opportunity to learn and enhance our resources, which can take various forms:
-
-- Best practices and guidelines in our documentation
-- Newsletters and blog posts sharing insights
-- Open source tools and libraries that solve common problems
-
-Our goal is to make Tuist the go-to reference for building apps at scale, providing the easiest and most effective path forward.
+Think of Tuist as something you seamlessly integrate into your repository and then forget.
```

Comment:
> Think of Tuist as something you seamlessly integrate into your repository and then forget.

`...and then forget` feels a bit weird – you probably don't forget Tuist is there as we expect developers to directly interact with it. I'd rephrase this sentence a bit.